### DISCUSSION TITLE
UA伪装支持（chromium浏览器视频硬解）

### DISCUSSION THREAD

**Discussion by ZerOri** [the1812/Bilibili-Evolved#2820] [功能建议 / Ideas]
之前看到一个为chromium浏览器启用HEVC硬解的教程 [Chrome/Egde上为B站启用HEVC硬解](https://www.bilibili.com/read/cv13253613 )，确实硬解能极大减少CPU和带宽压力（出租屋小水管耗不起），里面需要安装插件修改UA，所以请求脚本能支持UA伪装从而让B站支持硬解，省去安装插件的麻烦。

**Comment by the1812** [comment]
那是浏览器插件吧, 脚本改不了 UA
